src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -122,8 +122,8 @@ The .NET Foundation licenses this file to you under the MIT license.
 
     <ItemGroup>
       <NativeLibrary Condition="'$(IlcMultiModule)' == 'true'" Include="$(SharedLibrary)" />
-      <NativeLibrary Condition="'$(NativeLib)' == '' and '$(CustomNativeMain)' != 'true'" Include="$(IlcSdkPath)libbootstrapper.o" />
-      <NativeLibrary Condition="'$(NativeLib)' != '' or '$(CustomNativeMain)' == 'true'" Include="$(IlcSdkPath)libbootstrapperdll.o" />
+      <NativeLibrary Condition="'$(NativeLib)' == ''" Include="$(IlcSdkPath)libbootstrapper.o" />
```

Comment:
CustomNativeMain test is still failing. My guess is that it is due to this change. Should this condition stay as both here and in Microsoft.NETCore.Native.Windows.targets?

CustomNativeMain can be only ever usable with `dll` style bootstrapper, even when the final binary is .exe.

Reply:
Oh, I see. Reverted those changes in https://github.com/dotnet/runtime/pull/103504/commits/d469bbdc754c70a7c4503611fce6171701cc35a3 :)